libs/components/molecules/src/lib/footer/footer.component.ts

```diff
@@ -0,0 +1,86 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatIconModule } from '@angular/material/icon';
+import { MatButtonModule } from '@angular/material/button';
+import { ConnectedPosition, OverlayModule } from '@angular/cdk/overlay';
+import { MatListModule } from '@angular/material/list';
+
+/**
```

Comment:
All code documentation needs improvements. 'Download format interface' doesn't tell me what it is for and how it is used. Run compodoc and check the documentation generated (dist/compodoc/...)